docs/CloudflareWorkerReproduction.md

```diff
@@ -0,0 +1,178 @@
+# Reproducing ApplicationInsights Issues with Angular SSR in Cloudflare Workers
```

Comment:
Move all of the reproduction code to AISKU/Tests/CloudFlareWorkerRepro and update the docs